rasa/nlu/training_data/util.py

```diff
@@ -50,7 +50,7 @@ def get_file_format(resource_name: Text) -> Text:
     if fformat in [loading.MARKDOWN, loading.RASA_YAML] and all(
         f == fformat for f in file_formats
     ):
-        return fformat
+        return {loading.MARKDOWN: "md", loading.RASA_YAML: "yml"}[fformat]
```

Comment:
@degiz I'd rather modify the value of the `RASA_YAML` constant to `"yml"` (instead of `"rasa_yaml"`). Is there any reason we're making this distinction? As far as I know we didn't make the distinction for the markdown format. That's what causing the issues with `rasa data split nlu`

Reply:
Probably it's there to distinguish the `yml` as a file extension from the `yml` as a training data format, see `YAML_FILE_EXTENSIONS`.

Seems to me that this `get_file_format()` method, which is used only here btw, is broken by design, because from the usage it should return a file extension (defined in `rasa/data.py`) but in fact it return format name from `rasa/nlu/training_data/loading.py`.

Reply:
yes exactly 👍 I'll keep the constant value and cleanup the design of `get_file_format()`